### DISCUSSION TITLE
Confused about piper_train.export_onnx

### DISCUSSION THREAD

**Discussion by athomas404** [rhasspy/piper#228] [Q&A]
One of my buddies trained a model using piper, and sent me the checkpoint files. He was working on linux and I'm on windows though, so I got WSL2 up and running, duplicated the python environment as a linux-based python venv supporting flask (which will run piper subroutines), and pip installed  the piper venv. However, I've run into trouble with the export command. I get an error:

ImportError: cannot import name '_compare_version' from 'torchmetrics.utilities.imports' (/home/gus/piper/src/python/.venv/lib/python3.10/site-packages/torchmetrics/utilities/imports.py)

Is the install broken on WSL2 right now?

**Comment by rmcpantoja** [comment]
Hi @athomas404,
Please install torch lighting = 1.7.

**Comment by NadunSanjeevana** [comment]
Hi @athomas404 ,
The error message you're encountering regarding the torchmetrics library might be due to compatibility issues between your installed versions of libraries and PyTorch
